isis/src/mro/apps/hicubenorm/hicubenorm.cpp

```diff
@@ -162,6 +160,14 @@ namespace Isis {
                 }
             }
             }
+            
+            Isis::CubeAttributeOutput atts;
+            if (ui.WasEntered("FROM")) {
+                atts = ui.GetOutputAttribute("FROM");
```

Comment:
This looks like it should be getting the attributes from the TO parameter not the FROM parameter. Otherwise, a good catch.